### PR TITLE
Update Configuration-Properties.md

### PR DESCRIPTION
**Pull request by dgeo**
took me some hours to find out these changes !

### CODE REVIEW

File: docs/cas-mgmt-documentation/installation/Configuration-Properties.md [hunk 1]
```diff
@@ -41,11 +41,11 @@ The index `[0]` is meant to be incremented by the adopter to allow for distinct
 # mgmt.authz-attributes[0]=
 # mgmt.user-properties-file=classpath:user-details.properties
 
-# mgmt.services-repo=/etc/cas/services-repo
+# cas.service-registry.json.location=file:/etc/cas/services-repo
```
Comment (mmoayyed) [inline docs/cas-mgmt-documentation/installation/Configuration-Properties.md]: This is incorrect. Please revert this.

Comment (dgeo) [reply docs/cas-mgmt-documentation/installation/Configuration-Properties.md]: What is the good name ?

using mgmt.services-repo leads to:
```
***************************
APPLICATION FAILED TO START
***************************

Description:

Binding to target [Bindable@5b8814be type = org.apereo.cas.configuration.CasManagementConfigurationProperties, value = 'provided', annotations = array<Annotation>[@org.springframework.boot.context.properties.ConfigurationProperties(ignoreInvalidFields=false, ignoreUnknownFields=false, prefix="mgmt", value="mgmt")]] failed:

    Property: mgmt.services-repo
    Value: "/etc/cas/stest"
    Origin: "mgmt.services-repo" from property source "bootstrapProperties-casCompositePropertySource"
    Reason: The elements [mgmt.services-repo] were left unbound.

Action:

Update your application's configuration
```

Comment (dgeo) [reply docs/cas-mgmt-documentation/installation/Configuration-Properties.md]: But I see that  # cas.service-registry.json.location is not better (some code loads it, but changes are not made there, /etc/cas/services-repo is still used)

Comment (dgeo) [reply docs/cas-mgmt-documentation/installation/Configuration-Properties.md]: got it: mgmt.version-control.services-repo, I'll update this

**Pull request closed without merging**